components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -409,6 +409,18 @@ pub fn find_in_set(s: &Option<Bytes>, str_list: &Option<Bytes>) -> Result<Option
     })
 }
 
+#[rpn_fn]
+#[inline]
+pub fn trim_1_arg(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    Ok(arg.as_ref().map(|bytes| {
+        let s = String::from_utf8_lossy(bytes);
+        s.trim_start_matches(' ')
+            .trim_end_matches(' ')
```

Comment:
I think you can simply use `trim_matches`

Reply:
Good catch, done.